app/views/results/student/_student_summary.html.erb

```diff
@@ -135,24 +139,10 @@
       <strong><%= t('marker.marks.translated_to_points') %></strong>
       <span id='total_extra_percentage_as_points'><%= result.get_total_extra_percentage_as_points %></span><br>
     </div>
+    <% end %>
   <% end %>
 </div>
 
-<% unless @result.is_a_review? %>
```

Comment:
It's fine to remove the contents here, but then the condition `@result.is_a_review?` should be included above.